### PR TITLE
Add markdown editor for arguments

### LINKED ISSUE #45: Editor for arguments
Arguments support markdown, it makes sense to have some kind of editor like https://github.com/Ionaru/easy-markdown-editor

### PR DESCRIPTION
**Pull request by vsergiu93**
Closes #45

### CODE REVIEW

File: resources/views/components/argument-card/card.blade.php [hunk 1]
```diff
@@ -8,7 +8,21 @@
 
     <div class="grid gap-2 md:gap-4 w-full">
         @if(!$readonly && $isEditing?->is($argument))
-            <textarea wire:model="body" class="border-gray-200 rounded-lg" rows="5"></textarea>
+            <div>
+                <div wire:ignore>
+                <textarea id="argument-editor" x-init="
+                test = new EasyMDE({
```
Comment (vsergiu93) [inline resources/views/components/argument-card/card.blade.php:14]: Not sure I like the approach. I happy to look at some suggestions to clean this a bit.
Initially I tried to have this as a dedicated component which can be reused, but I wasn't able to figure out how this `wire:model` really works.

File: resources/views/rfc.blade.php [hunk 2]
```diff
@@ -2,6 +2,16 @@
     $metaImageUrl = action(\App\Http\Controllers\RfcMetaImageController::class, $rfc);
 @endphp
 
+@push('styles')
+    <link rel="stylesheet" href="https://unpkg.com/easymde/dist/easymde.min.css">
```
Comment (vsergiu93) [inline resources/views/rfc.blade.php:6]: Not sure yet if there is a better way to add this dependencies, I tried just installing the package via npm, but I wasn't able to initialize it.

Comment (brendt) [reply resources/views/rfc.blade.php:6]: Looks fine to me, but I'm a total frontend noob. So if anyone has a better suggestion they can let us know :)

**Comment by brendt** [comment]
Let me know when this is ready for review :)

File: resources/views/components/user-name.blade.php [hunk 3]
```diff
@@ -19,7 +19,7 @@ class="rounded-full shadow-sm w-[25px] h-[25px]"
         />
     @endif
 
-    <a href="{{ action(\App\Http\Controllers\PublicProfileController::class, $user) }}" class="group-hover:underline">
```
Comment (vsergiu93) [inline resources/views/components/user-name.blade.php:22]: For me this didn't work, I would get the following error: 
image

**Comment by vsergiu93** [comment]
I think it is ready for code review.

Thanks

File: resources/views/components/user-name.blade.php (hunk 3 above)
Comment (vsergiu93) [reply resources/views/components/user-name.blade.php]: NVM, I was missing latest migrations related to username.

**Comment by brendt** [comment]
Looks nice!

**Pull request merged**